networking/synthetic/stac-prepare-nodes.yaml

```diff
@@ -2,7 +2,48 @@
 - hosts: stac_nodes
   remote_user: root
   tasks:
-    
+
+  - name: install epel 7
+    yum:
+      name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+      state: present
+
+  - name: add rhel latest repos
+    yum_repository:
+      name: rhel_latest
+      description: RHEL perf-dept repo
+      baseurl: http://download.devel.redhat.com/rel-eng/latest-RHEL-7/compose/Server/x86_64/os/
```

Comment:
You can't put internal URLs in the svt repo.

Reply:
done